superset-frontend/src/common/components/index.tsx

```diff
@@ -163,17 +171,24 @@ export const StyledSubMenu = styled(AntdMenu.SubMenu)`
       background-color: ${({ theme }) => theme.colors.primary.base};
     }
   }
+  .ant-menu-submenu-arrow {
+    top: 67%;
+  }
   & > .ant-menu-submenu-title {
     padding: 0 ${({ theme }) => theme.gridUnit * 6}px 0
       ${({ theme }) => theme.gridUnit * 3}px !important;
     svg {
       position: absolute;
-      top: ${({ theme }) => theme.gridUnit * 4}px;
+      top: ${({ theme }) => theme.gridUnit * 4 + 7}px;
```

Comment:
Perhaps `top: ${({ theme }) => theme.gridUnit * 5 + 3}px;`?

Reply:
This depends on what relationship we want the svg to have to the grid unit. When pixel amounts are hardcoded, it should be to format the thing being positioned so that it can look right in any grid. We could also do `theme.gridUnit * 6 - 1`, for example.